.gitignore

```diff
@@ -1,52 +1,143 @@
-# Temporary Python files
-*.pyc
-*.egg-info
+.drafts
+config.json
 __pycache__
-.ipynb_checkpoints
-setup.py
+
+.vscode
+.secrets.json
+
+# CUSTOM FILES!
+
+images/
+playground/
+
+# CONFIG FILES
+mensa_config.py
+app/api/data/
+app/api/dummydata/speiseplan
```

Comment:
What is that? It looks like you added your personal gitignore file to this commit ?

Reply:
Yes, not personal, but a Boilerplate gitignore. 
ill remove those from the Boilerplate. 
- good catch!